payloads/library/ChromeOS/Chromebook-Demo/README.md

```diff
@@ -0,0 +1,24 @@
+# Chromebook Demo Payload
+
+ Author: Julz4455
+Version: 1.2
+
+ ## Description
+
+ Send a certain message to the victim of your hack.
+This can be a message to lock thier pc or to watch out because of an upcoming hack.
```

Comment:
*their, not thier

Reply:
Oops! Egg on my face. 😄 We can probably fix that later.